securedrop/source.py

```diff
@@ -217,12 +217,11 @@ def normalize_timestamps(sid):
     """
     sub_paths = [ store.path(sid, submission.filename)
                   for submission in g.source.submissions ]
-    if len(sub_paths) > 1:
-        args = ["touch"]
-        args.extend(sub_paths[:-1])
-        rc = subprocess.call(args)
-        if rc != 0:
-            app.logger.warning("Couldn't normalize submission timestamps (touch exited with %d)" % rc)
+    args = ["touch", "-t", datetime.now().strftime("%Y%m%d") + "0000"]
```

Comment:
Sweet, it's awesome that you can arbitrarily modify the last modified time like that. It might be worth filing a follow-up to make sure the actual creation time isn't being stored [somewhere sneaky](http://unix.stackexchange.com/questions/50177/birth-is-empty-on-ext4/50184#50184).